CarpetCleaning/Views/dbo.JobsOverview

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW [JobSpecifics] AS
+    SELECT * FROM (
```

Comment:
I don't think this outer SELECT * FROM is necessary.  Is it possible to get rid of it?

Reply:
Nope. When I get rid of it the pivot table does not work.

Reply:
Hmm, well I'm too lazy to look into this further, so I guess it's okay lol